CONTRIBUTING.md

```diff
@@ -1,67 +1,66 @@
 # Contributing
 
-This repository includes a [GitHub Action][4] that builds the
+This repository includes a [GitHub Action] that builds the
 colorschemes once a week. This keeps the colorschemes up-to-date
 automatically.
 
 ## Building
 
 ### Dependencies
 
-- `>=0.2.0` [base16-builder-go][1]
-- golang `>=1.16` to build base16-builder-go
+- `>=0.2.6` [base16-builder-go]
 
 ### Usage for template editing
 
-1. Clone [base16-builder-go][1] somewhere on your system.
-1. Run `cd /path/to/base-builder-go && go build` to generate a binary:
-`/path/to/base-builder-go/base16-builder-go`
-1. Now execute the binary you generated while giving the `-template-dir`
-arg the path to `base16-vim` repository: `./base16-builder-go
--template-dir ../base16-vim`
+1. Clone the template repository onto your system
+1. Download the [latest base16-builder-go binary]
+1. Execute the binary with the `-template-dir` arg and provide
+   `/path/to/base16-vim` 
 
 Or the above steps represented in shell commands:
```

Comment:
Here it would be nice to have copy pastable bash commands:
1. git clone
2. wget command which copies the binary to a .bin folder
3. ...